custom/enikshay/integrations/nikshay/field_mappings.py

```diff
@@ -105,7 +105,7 @@
 }
 
 smear_result_grade = {
-    '0': 99,  # Negative
```

Comment:
@mkangia 

What's the mapping for scanty?  We have a potential result grade of "scanty" and the scanty count is in max_bacilli_count.  Can we juse that for the mapping, or should we just sent 98?

Reply:
@sheelio 
We combine result grade scanty with bacilli count. So bacilli count can be between 0 and 9. So we combine them as "scanty-{bacilli_count}" to get the corresponding value.